package.json

```diff
@@ -14,40 +14,24 @@
     "copy:test-assets": "cpx \"./src/tests/assets/**/{*,.*}\" ./lib/tests/assets"
   },
   "dependencies": {
-    "@bentley/backend-itwin-client": "^2.17.2",
-    "@bentley/bentleyjs-core": "^2.17.2",
-    "@bentley/ecschema-metadata": "^2.17.2",
-    "@bentley/electron-manager": "^2.17.2",
-    "@bentley/frontend-authorization-client": "^2.17.2",
-    "@bentley/geometry-core": "^2.17.2",
-    "@bentley/imodelhub-client": "^2.17.2",
-    "@bentley/imodeljs-backend": "^2.17.2",
-    "@bentley/imodeljs-common": "^2.17.2",
-    "@bentley/imodeljs-frontend": "^2.17.2",
-    "@bentley/imodeljs-i18n": "^2.17.2",
-    "@bentley/imodeljs-quantity": "^2.17.2",
-    "@bentley/itwin-client": "^2.17.2",
-    "@bentley/orbitgt-core": "^2.17.2",
-    "@bentley/presentation-common": "^2.17.2",
-    "@bentley/product-settings-client": "^2.17.2",
-    "@bentley/rbac-client": "^2.17.2",
-    "@bentley/telemetry-client": "^2.17.2",
-    "@bentley/ui-abstract": "^2.17.2",
-    "@bentley/webgl-compatibility": "^2.17.2",
-    "electron": "^11.1.0"
+    "@itwin/insights-client": "0.4.0",
+    "@itwin/core-bentley": "^3.6.1",
+    "@itwin/core-backend": "^3.6.1",
+    "@itwin/node-cli-authorization": "^0.10.0",
+    "electron": "^23.1.2"
   },
   "devDependencies": {
-    "@bentley/build-tools": "^2.17.2",
-    "@bentley/eslint-plugin": "^2.17.2",
-    "@types/chai": "^4.2.21",
-    "@types/mocha": "^8.2.3",
+    "@bentley/build-tools": "2.19.51",
+    "@bentley/eslint-plugin": "2.19.51",
+    "@types/chai": "^4.1.4",
+    "@types/mocha": "^8.2.2",
     "@types/sinon": "^10.0.2",
-    "chai": "^4.3.4",
-    "eslint": "^6.8.0",
-    "mocha": "^9.0.2",
+    "chai": "^4.1.2",
+    "eslint": "^7.11.0",
+    "mocha": "^8.3.2",
```

Comment:
There are no tests in this app, so `mocha`, `chai`, `@types/chai`, `@types/mocha`, `@types/sinon`, `sinon` are not needed.

Reply:
Remove `test` and and `copy:test-assets` scrips as well.

Reply:
Remove `mocha` configuration from package.json.